crud/common/utils.lua

```diff
@@ -251,11 +254,57 @@ function utils.is_uuid(value)
     return ffi.istype(uuid_t, value)
 end
 
-function utils.format_result(rows, space)
-    return {
-        metadata = table.copy(space:format()),
-        rows = rows,
-    }
+local function get_field_metadata(full_metadata, field_name)
+    dev_checks('table', 'string')
+
+    local metadata = metadata_cache[full_metadata]
+    if metadata ~= nil then
+        return metadata[field_name]
+    end
+
+    metadata_cache[full_metadata] = {}
+    for _, field in ipairs(full_metadata) do
+        metadata_cache[full_metadata][field.name] = field
```

Comment:
Here you can see the difference between `field` and `field_name`, and variable name says that some variables are field objects (table with some field data), but others are `field_name` - a string with name of field.

But in `filter_format_fields` function `fields` is used as `field_names` and it confuses. 
I think that the only one place when we use `fields` name for field names array is CRUD functions API.
User says `crud.do_smth('space', {fields = {...}})`, we accept `opts.fields` here (because it's comfortable for user), but then we can use this value in internal functions as `field_names` to highlight that this is array of strings, not field objects (e.g. `*_on_storage` function can accept `field_names` since it's internal).